<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Продать книги</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font-family: Verdana, Geneva, sans-serif;
font-size: 12pt;</string>
         </property>
         <property name="text">
          <string>Введите кол-во книг которые вы хотите продать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="countEdit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="okBut">
         <property name="text">
          <string>Продать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="noBut">
         <property name="text">
          <string>Назад</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
